test_scripts/Defects/5_1/980_SDL_responds_GENERIC_ERROR_instead_of_INVALID_DATA_when_soft_button_has_Type_is_Image_or_Both_and_Text_is_whitespace_or_\t_or_\n_or_empty.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/980
+-- Description: [APIs] AlertManeuver: SDL responds GENERIC_ERROR instead of INVALID_DATA when soft button has Type is
```

Comment:
```suggestion
-- Description: [APIs] AlertManeuver: SDL responds GENERIC_ERROR instead of INVALID_DATA when soft button has Type
```

Reply:
@GetmanetsIrina, I think that no changes are required here, since there is a continuation on the next line